framework/src/auxkernels/FunctorAux.C

```diff
@@ -31,7 +31,8 @@ FunctorAux::FunctorAux(const InputParameters & parameters)
     _factor(getFunctor<Real>("factor")),
     _is_fe(dynamic_cast<MooseVariableFE<Real> *>(&_var))
 {
-  if (!_is_fe && !dynamic_cast<MooseVariableFV<Real> *>(&_var))
+  if (!_is_fe && !(dynamic_cast<MooseVariableFV<Real> *>(&_var) ||
+                   dynamic_cast<MooseLinearVariableFV<Real> *>(&_var)))
```

Comment:
this seems like a source of future mistakes.
should we just move _is_fe and add _is_fv in auxkernel.h